packages/server/src/schema/QueryType.ts

```diff
@@ -52,5 +57,15 @@ export default new GraphQLObjectType({
       resolve: async (_, args, context) =>
         StoreLoader.load(context, args.id),
     },
+    productsByStoreId: {
```

Comment:
avoid rest names

it is better get this from a node interface from Store then get a list of products inside of this store

Reply:
```
StoreType: {
  ...,
  products: {} // filter the products by storeId
}
```

then, on frontend you do a request by node interface